src/Kmd.Logic.Cpr.Client/IInternalClient.cs

```diff
@@ -81,7 +81,7 @@ internal partial interface IInternalClient : System.IDisposable
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
-        Task<HttpOperationResponse<object>> GetByIdWithHttpMessagesAsync(System.Guid subscriptionId, System.Guid id, System.Guid? configurationId = default(System.Guid?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
+        Task<HttpOperationResponse<Citizen>> GetByIdWithHttpMessagesAsync(System.Guid subscriptionId, System.Guid id, System.Guid? configurationId = default(System.Guid?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken));
```

Comment:
This change looks odd.

Reply:
Yes it is auto generated code based on swagger, I have changed swagger in PR 38224.

Reply:
Updated swagger and Fixed